### DISCUSSION TITLE
User "Tagging"

### DISCUSSION THREAD

**Discussion by ghost** [AntlerVC/firetable#851] [Support / Q&A] [answered]
What would be the best type of data for "tags"? Almost every CRM platforms have Tags and Labels to separate Users/Clients with individual tags and I am trying to figure out how to do the same thing with Firebase and Rowy tables. In my example, we will provide users to choose activities they have an interest in and choose from the list such as Football, Tennis, etc. Can be many activities. Each activity is a tag so we, as a platform, can show only content based on tags. If there is another option for a similar solution, which seems very basic to me, please let me know.

**Comment by shamsmosowi** [comment, marked as answer]
you can use multi-select field https://docs.rowy.io/field-types/supported-fields#select it will be stored as string array on firestore
